components/tools/OmeroWeb/omeroweb/settings.py

```diff
@@ -43,6 +43,16 @@
 from omero_ext import portalocker
 from omero.install.python_warning import py27_only, PYTHON_WARNING
 
+# Load custom settings from etc/grid/config.xml
```

Comment:
The comments in this block are no longer applicable. They should likely be left with the invoking code rather than with the imports.